charts/jaeger/templates/collector-svc.yaml

```diff
@@ -61,7 +61,7 @@ spec:
     targetPort: {{ .Values.collector.service.otlp.http.name }}
 {{- end }}
   - name: {{ .Values.collector.service.admin.name }}
```

Comment:
What was the meaning of "admin"? In v2 there is no single admin port.

Reply:
So what should I name it in `values.yaml`

Reply:
This specific port is healthcheck